qml/QtAcademyProjectForm.qml

```diff
@@ -29,4 +30,200 @@ Item {
         id: model
         mapView: view
     }
+
+    // A background rectangle over which to display the buttons
+    Rectangle {
+        id: background
+        anchors.fill: buttonCol
+        anchors.margins: -5
+        color: "white"
+        radius: 5
+    }
+
+    // The column to contain the button images
+    Column {
+        id: buttonCol
+        anchors {
+            top: parent.top
+            right: parent.right
+            margins: 15
+        }
+        spacing: 10
+
+        // Toggle between tracking the user's location with the line graphic
+        Image {
```

Comment:
you could look into using Button instead of Image with a MouseArea. Button should have the ability to set the icon, and  you can also make it checkable if you want it to go into a checked/unchecked state https://doc.qt.io/qt-6/qml-qtquick-controls-abstractbutton.html#icon.source-prop

Reply:
True. I like the look of the images on their own, but the buttons may be a little more straightforward.

Reply:
the nice thing about buttons is they give you some niceties built right in, such as visual feedback when your mouse hovers over, on click, etc